agent/ebs/watcher.go

```diff
@@ -156,14 +260,19 @@ func (w *EBSWatcher) notifyAttachedEBS(volumeId string) {
 		log.Warnf("State change event has already been emitted for EBS volume: %v.", ebs.EBSToString())
 		return
 	}
-
+	// We found an EBS volume which has the expiration time set in future and
+	// needs to be acknowledged as having been 'attached' to the Instance
+	if err := w.sendEBSStateChange(ebs); err != nil {
+		// skip logging status sent error as it's redundant and doesn't really indicate a problem
+		if !strings.Contains(err.Error(), ebsStatusSentMsg) {
```

Comment:
I don't see where `sendEBSStateChange` might return an error with `ebsStatusSentMsg` as its prefix. The only error it returns is `return errors.New("ebs watcher send EBS state change: nil volume")`.